packages/esm-patient-programs-app/src/programs/programs-detailed-summary.component.tsx

```diff
@@ -158,10 +153,12 @@ const ProgramsDetailedSummary: React.FC<ProgramsDetailedSummaryProps> = ({ patie
                   {rows.map((row, i) => (
                     <TableRow key={row.id}>
                       {row.cells.map((cell) => (
-                        <TableCell key={cell.id}>{cell.value?.content ?? cell.value}</TableCell>
+                        <TableCell className="cds--table-column-menu" key={cell.id}>
+                          {cell.value?.content ?? cell.value}
+                        </TableCell>
                       ))}
-                      <TableCell className="cds--table-column-menu">
-                        <ProgramEditButton programEnrollmentId={enrollments[i]?.uuid} t={t} />
+                      <TableCell>
```

Comment:
But it is here to ensure the OverflowMenu takes up the appropriate amount of space.

```suggestion
                      <TableCell className="cds--table-column-menu">
```